lms/static/js/verify_student/views/webcam_photo_view.js

```diff
@@ -314,6 +317,18 @@
                 .attr('aria-disabled', !isEnabled);
         },
 
+        setSubmitButtonFocused: function( ){
+            $( this.submitButton )
+                .trigger('focus');
+            $('#submit').attr('title', gettext( 'Photo Captured successfully '));
+        },
+
+        capture_sound: function(){
+            var audio = document.createElement('audio');
+            audio.src = '/static/audio/camera_capture.wav';
+            audio.play();
```

Comment:
@ahsan-ul-haq I think including it in the DOM via the template is the best approach. There is no need to remove the audio element, especially as it will be used multiple times because users need to take at least 2 pictures.